### PR TITLE
Add xclipboard, editorconfig and rust to Spacemacs

### PR DESCRIPTION
**Pull request by picandocodigo**

### CODE REVIEW

File: .spacemacs [hunk 1]
```diff
@@ -62,6 +62,7 @@ values."
      syntax-checking
      version-control
      personal-config
+     xclipboard
```
Comment (igelstorm) [inline .spacemacs]: Do you know if this would work for Mac?

Comment (picandocodigo) [reply .spacemacs]: I assume it will, but I tagged you and Phil to check it oot :grimacing:

Comment (igelstorm) [reply .spacemacs]: I'll try this branch on my machine and report back :)

Comment (igelstorm) [reply .spacemacs]: It does work on Mac! However, the `xclipboard` layer isn't yet in the Spacemacs master branch, which I was using. So to get this to work, I had to checkout the `develop` branch in my `~/.emacs.d`. We might need to mention this in the README?

Comment (picandocodigo) [reply .spacemacs]: Ah, yes. I did mention it in the README update but I assumed everyone was using `develop`.

Comment (igelstorm) [reply .spacemacs]: So you did 😆 I should probably have reviewed the entire PR... Never mind, that's fine then.

**Pull request merged**